### PR TITLE
CC-1809: changed post-stop to remove remaining isvcs containers on shutdown

### PR DESCRIPTION
**Pull request by smousa**
https://jira.zenoss.com/browse/CC-1809
Only need to look at first commit, because I was fixing the spacing of the file in the second commit.

### CODE REVIEW

File: pkg/serviced-systemd.sh [hunk 1]
```diff
@@ -6,39 +6,38 @@ rc=0
...
+            sleep 5
+        done
+        
+        # wait for serviced to stop listening
+        echo "$(date): waiting for serviced to stop listening"
+        while netstat -plant | grep 'LISTEN.*/serviced$'; do
+            sleep 2
+        done
+        
+        # stop and remove potentially running isvcs even though serviced stopped (column 2 is IMAGEID)
```
Comment (tgjonestx) [inline pkg/serviced-systemd.sh]: All of these changes need to be applied to the upstart script for Ubuntu

Comment (smousa) [inline pkg/serviced-systemd.sh]: Good catch, I forgot about that

**Comment by smousa** [comment]
retest this

**Comment by smousa** [comment]
retest this

**Pull request merged**